### PR TITLE
 DDF-5168 Upgrade CXF to 3.2.9 

### PR DESCRIPTION
**Pull request by stustison**
#### Who is reviewing it? 
@bakejeyner @SmithJosh 

#### Ask 2 committers to review/merge the PR and tag them here.

@blen-desta
@garrettfreibott
@stustison 

#### What are the relevant tickets?
Fixes: #____

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by stustison** [comment]
build now

**Comment by stustison** [comment]
build now

**Comment by stustison** [comment]
build now

**Comment by stustison** [comment]
build now

**Comment by stustison** [comment]
build now

**Comment by stustison** [comment]
build now

**Comment by stustison** [comment]
build now

**Comment by stustison** [comment]
build now

**Comment by stustison** [comment]
build now

**Comment by stustison** [comment]
build now

**Comment by stustison** [comment]
build now

**Comment by stustison** [comment]
build now

**Comment by stustison** [comment]
build now

**Comment by stustison** [comment]
build now

**Comment by stustison** [comment]
build now

**Comment by stustison** [comment]
build now

**Comment by stustison** [comment]
build now

**Comment by stustison** [comment]
build now

**Comment by stustison** [comment]
build now

**Comment by stustison** [comment]
build now

**Comment by stustison** [comment]
build now

**Comment by stustison** [comment]
build now

File: distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java [hunk 1]
```diff
@@ -552,7 +552,8 @@ private static Integer findPortNumber(Integer portToTry) {
                 + "http://svn.apache.org/repos/asf/servicemix/m2-repo@id=servicemix,"
                 + "http://repository.springsource.com/maven/bundles/release@id=springsource,"
                 + "http://repository.springsource.com/maven/bundles/external@id=springsourceext,"
-                + "http://oss.sonatype.org/content/repositories/releases/@id=sonatype"),
+                + "http://oss.sonatype.org/content/repositories/releases/@id=sonatype,"
+                + "http://artifacts.codice.org/content/repositories/releases"),
```
Comment (LinkMJB) [inline distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java]: I think we might need to include http://artifacts.codice.org/content/repositories/thirdparty in order to get the desired artifacts to resolve properly

**Comment by stustison** [comment]
build now

**Comment by stustison** [comment]
build now

**Pull request merged**